lib/hydra/derivatives/runners/active_encode_derivatives.rb

```diff
@@ -0,0 +1,39 @@
+module Hydra::Derivatives
+  class ActiveEncodeDerivatives < Runner
+    # @param [String, ActiveFedora::Base] object_or_filename source file name (or path), or an object that has a method that will return the file name
+    # @param [Hash] options options to pass to the encoder
+    # @option options [Symbol] :source a method that can be called on the object to retrieve the source file's name
+    # @option options [Symbol] :encode_class class name of the encode object (usually a subclass of ::ActiveEncode::Base)
+    # @options options [Array] :outputs a list of desired outputs
+    def self.create(object_or_filename, options)
+      processor_opts = processor_options(options)
+      source_file(object_or_filename, options) do |file_name|
+        transform_directives(options.delete(:outputs)).each do |instructions|
+          processor = processor_class.new(file_name, instructions, processor_opts)
+          processor.process
+        end
+      end
+    end
+
+    def self.processor_options(options)
```

Comment:
`.processor_options` is the only method here that isn't an override of a public method in `Runner`.  I'll move it to be private.